.github/pull_request_template.md

```diff
@@ -1,27 +1,21 @@
 # Objective
 
-- Describe the objective or issue this PR addresses.
-- If you're fixing a specific issue, say "Fixes #X".
+Why are you making these changes?
+Link any associated issues with "Fixes #X" so they can be automatically closed.
+If this is a particularly complex PR, please add a clear list of the changes made.
 
-## Solution
+Discuss any background information that may be needed for a new contributor to understand the changes you've made.
```

Comment:
I wouldn't call out "new contributor" as people who understand less

Reply:
I would rather be less explicit, with "other contributors", or "reviewers". context and background is always good, not just for people unfamiliar